app/models/generic_phrase.rb

```diff
@@ -2,4 +2,24 @@ class GenericPhrase < ApplicationRecord
   belongs_to :verb
   belongs_to :adjective
   has_many :phrase_generic_phrases, dependent: :destroy
+
+  def self.search(start_date, end_date, options = {})
+    options = {
```

Comment:
Nice!

Reply:
Yeah, gets around the problem of supplying empty keyword parameters which then take priority over default values.